.gitignore

```diff
@@ -0,0 +1 @@
+index.html
```

Comment:
@phochste this prevents the build html from being accessble. I made this mistake before ;)

Reply:
ai ai ai